tests/baselines/reference/user/enhanced-resolve.log

```diff
@@ -32,47 +17,14 @@ node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(209,4): error TS2322:
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(220,4): error TS2322: Type 'null' is not assignable to type '(path: any) => any'.
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(224,23): error TS2322: Type 'null' is not assignable to type '(path: any, callback: any) => void'.
 node_modules/enhanced-resolve/lib/CachedInputFileSystem.js(227,27): error TS2322: Type 'null' is not assignable to type '(path: any) => any'.
-node_modules/enhanced-resolve/lib/CloneBasenamePlugin.js(7,40): error TS2339: Property 'basename' does not exist on type '(path: any) => { [x: string]: any; paths: any[]; seqments: any[]; }'.
-node_modules/enhanced-resolve/lib/CloneBasenamePlugin.js(20,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ConcordExtensionsPlugin.js(26,24): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ConcordMainPlugin.js(26,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ConcordModulesPlugin.js(29,30): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ConcordModulesPlugin.js(34,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js(32,24): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/FileKindPlugin.js(17,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/JoinRequestPlugin.js(16,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/MainFieldPlugin.js(44,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ModuleAppendPlugin.js(31,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ModuleKindPlugin.js(17,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ModulesInHierachicDirectoriesPlugin.js(30,26): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ModulesInRootPlugin.js(17,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/ParsePlugin.js(17,23): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
-node_modules/enhanced-resolve/lib/Resolver.js(19,26): error TS2304: Cannot find name 'Map'.
-node_modules/enhanced-resolve/lib/Resolver.js(160,29): error TS2304: Cannot find name 'Set'.
+node_modules/enhanced-resolve/lib/Resolver.js(162,17): error TS2339: Property 'push' does not exist on type 'Set<any>'.
```

Comment:
fun stuff.. used to be an array, now a set, just put a `push` method on it, and everybody is happy.